React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -128,6 +132,9 @@ - (void)presentViewController:(UIViewController *)alertController
                                                       handler:^(__unused UIAlertAction *action) {
                                                         callback(@[ @(localIndex) ]);
                                                       }]];
+    if ([disabledButtonIndices containsObject:@(localIndex)]) {
```

Comment:
not a biggie, but instead of asking if the action is disabled _for each of the actions_, you can iterate over indices in  `disabledButtonIndices` and call `setEnabled:false` for _each index you know is disabled_.

Reply:
Done! Thanks for suggestion.

Reply:
now that I'm looking at it, the problem you didn't have before but have now is handling the situation when disabledButtonIndices contains index that is out of bounds.

Reply:
Disable button if its _index_ is within the allowable range, otherwise throw an error with a message.